Add recent trades

dcts opened the pull request:
Added a tab inside the orderbook panel to show "recent trades" of the pair. 

# Preview

![image](https://github.com/user-attachments/assets/17093ac1-3706-48b7-a95e-b3cbc843b2e4)

fliebenberg commented:
I have replied on Discord. I think it might be better to import the trades in the same way other adex variables are imported in the subscriptions.ts file.

dcts commented:
@fliebenberg thanks for the pointers, I added the state to the orderBookSlice and updated via subscriptions. Now the data should not be static anymore but updating in realtime! :)
